### PR TITLE
Add song

### LINKED ISSUE #16: 8 - Add songs
When I, as scout leader, prepare for a scout event, I want to easily add a new song to the songbook, so that all the scouts have access to it.

### PR DESCRIPTION
**Pull request by amunds1**
Implementert:
- Formatert iht eksempler sendt fra DF
- Lagt til chorus i Sanity
- Visning av riktig sangformat

Mobprogget med @amunds1 @KristofferNyvoll @Lekesoldat 

Closes #16

### CODE REVIEW

File: frontend/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,856 @@
+schema {
```
Comment (Chaaarles) [inline frontend/schema.graphql:1]: This file looks to be automatically generated. Do we need it in the repo, or could its generation be added as an installation step?

**Review by Chaaarles — COMMENTED** [review]
Code looks good. Added a comment as food for thought

**Comment by Chaaarles** [comment]
Får opp noen feilmeldinger når jeg navigerer til sangene
image

File: frontend/src/components/molecules/VerseNumbering.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+const VerseNumbering: FC<VerseNumberingProps> = ({ song: { verses } }) => (
+  <Stack spacing={6} h="100%">
+    {verses.map((verse, i) => (
+      <>
+        <Text key={`verse${i}`} whiteSpace="pre-wrap">
+          {`${i + 1}. ${verse}`}
+        </Text>
+      </>
```
Comment (Chaaarles) [inline frontend/src/components/molecules/VerseNumbering.tsx:16]: Remove fragment tag. 

```suggestion
        <Text key={`verse${i}`} whiteSpace="pre-wrap">
          {`${i + 1}. ${verse}`}
        </Text>
```

File: frontend/src/components/molecules/VerseChorusNumbering.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+}) => (
+  <Stack spacing={6} h="100%">
+    {verses.map((verse, i) => (
+      <>
+        <Text key={`verse${i}`} whiteSpace="pre-wrap">
+          {`${i + 1}. ${verse}`}
+        </Text>
+
+        {i === 0 && <Text>{`Ref: ${chorus}`}</Text>}
+      </>
```
Comment (Chaaarles) [inline frontend/src/components/molecules/VerseChorusNumbering.tsx:20]: The fragment tag should be removed.
React requires a `key` prop for every element returned in a map, and this fragment does not have a that

**Review by Chaaarles — CHANGES_REQUESTED** [review]

File: frontend/schema.graphql (hunk 1 above)
Comment (kharann) [reply frontend/schema.graphql:1]: should be deleted probably

File: frontend/src/components/molecules/VerseChorusNumbering.tsx (hunk 3 above)
Comment (Lekesoldat) [reply frontend/src/components/molecules/VerseChorusNumbering.tsx]: Oppdaterte den nå. La til key på fragmenten i stedet. Trenger fragment for å rendre mer enn ett element, som vi gjør på linje 19 dersom et chorus er lagt ved.

Comment (Chaaarles) [reply frontend/src/components/molecules/VerseChorusNumbering.tsx]: Bra løsning!

File: frontend/schema.graphql (hunk 1 above)
Comment (Chaaarles) [reply frontend/schema.graphql:1]: The frontend appears to run fine without it.
Any specific reason it was added, @amunds1 @Lekesoldat @KristofferNyvoll?

Comment (kharann) [reply frontend/schema.graphql:1]: solved in another PR actually

**Review by Chaaarles — APPROVED** [review]
🔫🤠👉

**Pull request merged**